docs/content/en/docs/tasks/cluster/cluster-replace-cilium.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Replace EKS Anywhere Cilium with a custom CNI"
+linkTitle: "Add custom CNI"
+weight: 21
+description: >
+ Replace EKS Anywhere Cilium with a custom CNI
+---
+
+## Prerequisites
+
+* EKS Anywhere v0.15+.
+* A recent version of [Cilium CLI](https://github.com/cilium/cilium-cli).
+
+## Add a custom CNI to a new cluster
```

Comment:
We should also document the steps to pause/unpause the cluster during cluster operations.
Here's the steps for ref
Steps to pause the MachineHealthChecks >>
------------------------------------------------
```
kubectl --kubeconfig=${MANAGEMENT_KUBECONFIG} -n eksa-system annotate clusters.cluster.x-k8s.io   ${WORKLOAD_CLUSTER_NAME} cluster.x-k8s.io/paused=true  
kubectl --kubeconfig=${MANAGEMENT_KUBECONFIG} -n eksa-system annotate machinehealthchecks ${WORKLOAD_CLUSTER_NAME}-kcp-unhealthy cluster.x-k8s.io/paused=true  
kubectl --kubeconfig=${MANAGEMENT_KUBECONFIG} -n eksa-system annotate machinehealthchecks ${WORKLOAD_CLUSTER_NAME}-md-0-worker-unhealthy cluster.x-k8s.io/paused=true  
```
Steps to unpause the MachineHealthChecks >>
----------------------------------------------
```
kubectl --kubeconfig=${MANAGEMENT_KUBECONFIG} -n eksa-system annotate clusters.cluster.x-k8s.io   ${WORKLOAD_CLUSTER_NAME} cluster.x-k8s.io/paused-  
kubectl --kubeconfig=${MANAGEMENT_KUBECONFIG} -n eksa-system annotate machinehealthchecks ${WORKLOAD_CLUSTER_NAME}-kcp-unhealthy cluster.x-k8s.io/paused-  
kubectl --kubeconfig=${MANAGEMENT_KUBECONFIG} -n eksa-system annotate machinehealthchecks ${WORKLOAD_CLUSTER_NAME}-md-0-worker-unhealthy cluster.x-k8s.io/paused-  
```

Reply:
Thanks, I forgot about this. I'll add some cross references from cni.md to this as well from the warning.